quint/src/runtime/impl/compilerImpl.ts

```diff
@@ -1418,8 +1432,8 @@ export class CompilerVisitor implements IRVisitor {
                   // the run. Hence, do not report an error here, but simply
                   // drop the run. Otherwise, we would have a lot of false
                   // positives, which look like deadlocks but they are not.
-                  this.execListener.onUserOperatorReturn(nextApp, [], nextResult)
                   break
+                  this.execListener.onUserOperatorReturn(nextApp, [], nextResult)
```

Comment:
Seems like this line is not reachable anymore. Should we remove it?
```suggestion
```

Reply:
Oh, it should actually be before `break`. Thanks for finding that!